api/v3/VolunteerUtil.php

```diff
@@ -223,13 +203,20 @@ function _volunteerGetProjectRelationshipDefaults() {
       'relationship_type_id' => $employerRelationshipTypeId,
       'is_active' => 1,
     ));
-    $defaultBeneficiary = array($result);
+    $defaultBeneficiary = array('contact_id' => $result);
   } catch(Exception $e) {
     $domain = civicrm_api3('Domain', 'getsingle', array('current_domain' => 1));
-    $defaultBeneficiary = array($domain['contact_id']);
+    $defaultBeneficiary = array('contact_id' => $domain['contact_id']);
   }
   $defaults[$beneficiaryType] = $defaultBeneficiary;
 
+  foreach ($defaults as $type => $default) {
+    $defaults[$type]['can_be_read_by_current_user'] = CRM_Volunteer_BAO_ProjectContact::contactIsReadable($default['contact_id']);
```

Comment:
This doesn't allow the defaults to have more than one contact per relationship type. For now, I think that works, but when we allow defaults to be edited by the user this would need to be a nested foreach like I originally had.